jobs/config.json

```diff
@@ -4448,16 +4448,18 @@
   },
   "ci-kubernetes-e2e-gci-gke": {
     "args": [
+      "--check-leaked-resources",
+      "--cluster=",
```

Comment:
The idea is to let boskos send busted projects for developer to take a look, but it's kinda hard to define a 'busted project'.